pom.xml

```diff
@@ -102,20 +125,22 @@
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-classic</artifactId>
 			<version>1.2.3</version>
+			<scope>runtime</scope>
 		</dependency>
 
 		<!-- test dependencies -->
+
 		<dependency>
 			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-engine</artifactId>
-			<version>${junit.jupiter.version}</version>
+			<artifactId>junit-jupiter-api</artifactId>
```

Comment:
You replaced the engine with the API, have you checked the build if the corresponding tests are still being executed?

I recommend to use the include-all dependency: `junit-jupiter`.

Reply:
@beatngu13 I am discussing all of this together with @githubert - please view this PR rather as a draft PR, I will change the status accordingly

Reply:
OK, thx. Happened to me once: https://github.com/retest/recheck/issues/186/